parent_models_to_solr_field.xslt

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:string="xalan://java.lang.String"
+  xmlns:encoder="xalan://java.net.URLEncoder">
+
+  <xsl:template name="perform_ri_query">
+    <xsl:param name="PID"/>
+    <xsl:param name="risearch">http://localhost:8080/fedora/risearch</xsl:param>
+    <xsl:param name="query"/>
+    <xsl:param name="lang">sparql</xsl:param>
+    <xsl:param name="additional_params"/>
+
+    <xsl:variable name="encoded_query" select="encoder:encode(normalize-space(string:replaceAll($query, '%PID%', $PID)))"/>
+    <xsl:variable name="query_url" select="concat($risearch, '?query=', $encoded_query, '&amp;lang=', $lang, $additional_params)"/>
+
+    <xsl:copy-of select="document($query_url)"/>
+  </xsl:template>
+
+  <xsl:template name="get-parent-models">
+    <xsl:call-template name="perform_ri_query">
```

Comment:
Should... this perform a full traversal? I know the ticket descriptions specifies "children", but, is this naive? Should it be "descendants"?

Say a collection is contained within one of these objects... should we show the children of this contained collection in results?... _ramble ramble ramble_